blogposts/2021/api-documentation-generated-for-all-your-code.md

```diff
@@ -0,0 +1,84 @@
+- [Explore Go net/http code](https://sourcegraph.com/github.com/golang/go/-/docs/net/http)
+- [Explore Sourcegraph backend code](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/docs/cmd/frontend/backend)
+
+Want to try it out on your own Go repositories? Tweet [@sourcegraph](https://twitter.com/sourcegraph) with your GitHub repository name and we'll set it up for you, or [follow our documentation]([https://docs.sourcegraph.com/code_intelligence/apidocs](https://docs.sourcegraph.com/code_intelligence/apidocs)).
+
+## Improving rapidly
+
+Here's a little highlight of the changes we've made recently.
+
+### On by default, for everyone
+
+The most important announcement here is that API docs is now an on-by-default experiment for everyone! You can begin trying it out on Sourcegraph.com as well as your own Sourcegraph instance by clicking on the API docs button on any repository page:
+
+![API Docs button](https://sourcegraphstatic.com/blog/api-docs/api-docs-button.png)
+
+
+Once you click it, you'll be greeted with information about how to get started in a brand new welcome prompt we’ve added:
+
+![Welcome prompt](https://sourcegraphstatic.com/blog/api-docs/welcome-prompt.png)
+
+### Better sub-page navigation
+
+Within a Go package, there are many sub-packages. We've improved the way we render documentation subpages so you can skip over useless directories and get right to the content (but we're still hard at work improving navigation in API docs!):
+
+![Sub-page navigation](https://sourcegraphstatic.com/blog/api-docs/subpage-navigation.gif)
+
+### Understands the types of symbols, and uses iconography to communicate
+
+Due to API docs being built on code intelligence data, it understands exactly what symbols are and has detailed information about them. Whether symbols are deprecated, whether they're a function, variable, interface, etc. and the UI now communicates this information using icons:
+
+![Symbols information](https://sourcegraphstatic.com/blog/api-docs/symbols.gif)
+
+### A deep understanding of code
+
+We're paying attention to the small details, and really tailoring API docs to each language. We want an experience that makes sense for Go first, then we'll expand to more languages and build an experience that makes sense for another language.
+
+One way we've enabled this is by allowing LSIF language indexers to emit data in the way they see fit. Sourcegraph merely renders it, indexes it, and adds a nice navigation experience. For example, the Go LSIF indexer even chooses to display URL hashes in a way that makes sense specifically for Go, with a `Receiver.Method` format:
+
+![URL hash display for Go](https://sourcegraphstatic.com/blog/api-docs/url-hash-displays.gif)
+
+And it understands symbol types, too—just displaying them for now, but enabling you to search and view e.g. "just variables", "just functions", etc. in the near future:
```

Comment:
How is this different to what you describe in `Understands the types of symbols, and uses iconography to communicate` above (60-64)?

Reply:
Yeah, you're right, it's pretty duplicative actually. The point here is more about "We'll use this to bring you some cooler features in the future" - how about this:

```suggestion
In the near future, we'll enable you to search and filter based on symbol types as well. "just show me variables", "just private and deprecated functions", etc. based on the deep understanding of the code we've built:
```